### PR TITLE
data:AdvaitJain

### PR DESCRIPTION
**Pull request by Zoronium**

### CODE REVIEW

File: data/Advait_Jain.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "name": "Discord Community",
+      "url": "https://discordapp.com/users/759139442493095947",
+      "icon": "FaDiscord"
+    },
+    {
+      "name": "LinkedIn Organisation",
+      "url": "https://www.linkedin.com/in/advaitj",
+      "icon": "FaLinkedin"
+    }
+  ],
```
Comment (kumarsonsoff3) [inline data/Advait_Jain.json:27]: Please remove this trailing comma from here. 
```suggestion
  ]
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Your filename must be exact same as your username. So Please update it to `Zoronium.json` [here](https://github.com/Zoronium/LinkFree/edit/Zoronium-patch-1/data/Advait_Jain.json). Also, make the changes requested below. 🛠

**Review by accodes21 — CHANGES_REQUESTED** [review]
Change filename to your github username. Also remove the last unnecessary comma.

File: data/Advait_Jain.json (hunk 1 above)
Comment (Zoronium) [reply data/Advait_Jain.json]: done

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**